README.rst

```diff
@@ -0,0 +1,484 @@
+
+
+
+vital
+=====
+
+Vital is an open source C++ collection of libraries and tools that
+supply basic types and services to the Kitware KWIVER imagery tool
+kit.
+
+Overview of Directories
+-----------------------
+
+* CMake -- contains CMake helper scripts
+* tests -- contains testing related support code
+* vital -- contains the core library source and headers
+* vital/algo -- contains abstract algorithm definitions
+* vital/bindings -- contains C and Python bindings
+* vital/config -- contains configuration support code
+* vital/exceptions -- contains the exception class hierarchy
+* vital/io -- contains the classes that support reading and writing core data types
+* vital/kwiversys -- contains the code that supports the OS abstraction layer
+* vital/logger -- contains the classes that provide logging support
+* vital/plugin_loader --   contains the classes that provide plugin loading services
+* vital/tests -- contains the main testing code
+* vital/tools -- contains source for command line utilities
+* vital/types -- contains the source for the core data types
+* vital/util --   contains the source for general purpose utilities
+* vital/video_metadata -- contains the classes that support video metadata
+
+
+Running CMake
+-------------
+
+We recommend building kwiver out of the source directory to prevent
+mixing source files with compiled products.  Create a build directory
+in parallel with the kwiver source directory.  From the command line,
+enter the empty build directory and run::
+
+    $ ccmake /path/to/kwiver/source
+
+where the path above is the location of your kwiver source tree.  The
+ccmake tool allows for interactive selection of CMake options.
+Alternatively, using the CMake GUI you can set the source and build
+directories accordingly and press the "Configure" button.
+
+
+CMake Options
+=============
+
+* CMAKE_BUILD_TYPE -- The compiler mode, usually Debug or Release
+* CMAKE_INSTALL_PREFIX -- The path to where you want the kwiver build products to install
+* KWIVER_BUILD_SHARED -- Build shared or static libraries
+* KWIVER_ENABLE_DOCS -- Turn on building the Doxygen documentation
+* KWIVER_ENABLE_LOG4CLUS -- Enable log4cplus logger back end
+* KWIVER_ENABLE_PYTHON -- Enable the python bindings
+* KWIVER_ENABLE_TESTS -- Build the unit tests
+* KWIVER_USE_BUILD_TREE -- When building the plugin manager, whether to include the build directory in the search path.
+* KWIVER_ENABLE_C_BINDINGS -- Whether to build the C bindings
+* fletch_DIR -- Build directory for the Fletch support packages.
+
+There are many more options
+
+Dependencies
+------------
+
+Vital has minimal required dependencies. Sprokit pipeline framework
+relies on boost.  Arrows and sprokit processes are structured so that
+the code that depends on an external package is in a directory with
+the major dependency name (e.g. vxl, ocv). The dependencies can be
+turned ON or OFF through CMake variables.
+
+Development
```

Comment:
I would move this to a separate doc and link from here.  It's often called "CONTRIBUTING.rst".